ROADMAP.md

```diff
@@ -29,28 +29,24 @@ new Federation 2 (on the [main branch](https://github.com/apollographql/federati
 
 ## What's Next
 
-### Next Release
-
-* `@tag` export to API schema
-* Compose user-defined directives in subgraphs
-
 ### Under Consideration
 
-* `@defer` support in query planning
-* Subscriptions support in query planning
+* Ability to add fields to one subgraph at a time, allowing those fields to be immediately usable by clients. 
```

Comment:
Nit: I'd slightly modify to "Ability to add *value type* fields to one subgraph at a time, ...". I think without that context someone may be confused to what problem we're talking about since this is not an issue at all for entity types for instance.

Reply:
Thank you! Went ahead and added.